packages/experiments/src/demo/index.tsx

```diff
@@ -36,9 +37,10 @@ function _onLoad() {
 
   ReactDOM.render(
     <Fabric>
-      <Router onNewRouteLoaded={ _scrollAnchorLink }>
+      {/*<Router onNewRouteLoaded={ _scrollAnchorLink }>
```

Comment:
"-- No intention to merge in this format, want to start a discussion, will add some more details later --"